NEWS.md

```diff
@@ -1,4 +1,28 @@
-# mrgsolve (development version)
+# mrgsolve 1.5.0
+
+- New functions `mwrite_yaml()` and `mwrite_cpp()` can write a model object back
+  to a file, accounting for all updates since the model was read from native
+  mrgsolve format using `mread()` (#1213).
+  
+- New function `mread_yaml()` for reading back models written out with 
+  `mwrite_yaml()` (#1213).
```

Comment:
I think these should also list #1190 (or even only listing #1190 would be fine given #1213 was on offshoot of #1190).